DEVELOPMENT.md

```diff
@@ -99,6 +99,8 @@ docker rm piper
 The steps are generated based on the yaml files in `resources/metadata/` with the following command
 `go run pkg/generator/step-metadata.go`.
 
+Alternatively you can use `go generate` in directory `cmd/`
```

Comment:
I would be in favour of dropping the other way for carities sake. If `go generate` is the way to do it, let's not confuse the reader by useless synonyms.

Reply:
I removed the previous way of calling it.